chaos.py

```diff
@@ -1,34 +1,22 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import time
 import os
+from os.path import dirname, abspath, join
+
+import time
 import sys
 import logging
-import threading
-import http.server
-import random
 import subprocess
 import settings
-import patch
 import schedule
 
-from os.path import dirname, abspath, join
-
 import cron
 import github_api as gh
-import github_api.prs
-import github_api.voting
-import github_api.repos
-import github_api.comments
-
-# Has a sideeffect of creating private key if one doesn't exist already
-import encryption
```

Comment:
While this isn't currently used, it probably will be in the near future. I'd also like to preserve that side effect for local testing.